packages/git.rb

```diff
@@ -3,39 +3,40 @@
 class Git < Package
   description 'Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.'
   homepage 'https://git-scm.com/'
-  version '2.19.0'
-  source_url 'https://github.com/git/git/archive/v2.19.0.tar.gz'
-  source_sha256 'c4146d94d1c74de0d099bcd6ba22cc44f799332a46146ede4ad9c9f67971aa28'
+  version '2.20.1'
+  source_url 'https://github.com/git/git/archive/v2.20.1.tar.gz'
+  source_sha256 'a3fe1d35b00ec2e48f21f690d4d4b2b061ff132cba3d68684c530b12c8ef227f'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/git-2.19.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/git-2.19.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/git-2.19.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/git-2.19.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'c8ed38f5e4e8ce268f42c0f65c47b8a18674c9704ed8a9eefb376aedb60ced69',
-     armv7l: 'c8ed38f5e4e8ce268f42c0f65c47b8a18674c9704ed8a9eefb376aedb60ced69',
-       i686: '3c0e0b6fcfd0c07b2722c6c89998133281a49a5d265b3fb7f3b5309247909bf3',
-     x86_64: '80cc91e879dde9578668ea091a5d5f3129b57231b334bc4aa4cf4a5739f4feee',
   })
 
   depends_on 'curl' => :build
   depends_on 'python27' => :build     # requires python2
 
   # need to build using single core
-  @make_cmd = "make -j1 prefix=#{CREW_PREFIX} CC=gcc PERL_PATH=#{CREW_PREFIX}/bin/perl PYTHON_PATH=#{CREW_PREFIX}/bin/python2"
+  @make_cmd = "make -j1"
 
   def self.build
-    system "autoconf"
-    system "./configure --without-tcltk"
+    system "autoreconf -i"
+    system "./configure",
+           "--without-tcltk",
+           "--prefix=#{CREW_PREFIX}",
+           "--libdir=#{CREW_LIB_PREFIX}",
+           "--with-perl=#{CREW_PREFIX}/bin/perl",
+           "--with-pager=#{CREW_PREFIX}/bin/less",
+           "--with-editor=#{CREW_PREFIX}/bin/nano",
+           "--with-python=#{CREW_PREFIX}/bin/python2",
+           "--with-gitconfig=#{CREW_PREFIX}/etc/gitconfig",
+           "--with-gitattributes=#{CREW_PREFIX}/etc/gitattributes"
     system "#{@make_cmd} all"
   end
 
   def self.install
     system "#{@make_cmd} DESTDIR=#{CREW_DEST_DIR} install"
-    system "mkdir -p #{CREW_DEST_PREFIX}/share/git-completion"
-    system "cp -r contrib/completion/* #{CREW_DEST_PREFIX}/share/git-completion"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/git-completion"
+    system "cp -a contrib/completion/. #{CREW_DEST_PREFIX}/share/git-completion/"
```

Comment:
Why does this need changed?

Reply:
`FileUtils.mkdir_p "` takes advantage of Ruby more than `system "mkdir -p `.
`cp -a...` preserves file permissions.
Using '.' instead of '*' also copies dotfiles (Files with names starting with '.').

Reply:
Without the `-r` option, it won't recursively copy directories.  If there are no directories, then this is alright.

Reply:
`-a` implies `-R`:
```
  -a, --archive                same as -dR --preserve=all
  -R, -r, --recursive          copy directories recursively
      --preserve[=ATTR_LIST]   preserve the specified attributes (default:
                                 mode,ownership,timestamps), if possible
                                 additional attributes: context, links, xattr,
                                 all
```

Reply:
News to me.